docs/antora.yml

```diff
@@ -19,7 +19,7 @@ asciidoc:
     # All page-latest attributes are used as a variable in the docs for things like download links and search results
     page-latest-cli: '5.2021.09'
     # Must be lowercase because this is how the version appears in the docs
-    page-latest-supported-mc: '5.5.0'
+    page-latest-supported-mc: '5.5'
```

Comment:
It might be worth investigating how this happened in the first place in case we have some misconfigured release automation that will break stuff in future.